src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -371,6 +371,9 @@ def acs_get_credentials(dns_prefix, location):
     home = os.path.expanduser('~')
```

Comment:
I know this is not part of this PR, but I'd consider renaming the ```acs_get_credentials``` to something that include the orchestrator (Kubernetes) as it's only specific to that.